tardis/io/tests/data/csvy_test_config.yml

```diff
@@ -0,0 +1,77 @@
+# Example YAML configuration for TARDIS
+tardis_config_version: v1.0
+
+supernova:
+  luminosity_requested: 8.770 log_lsun
+  time_explosion: 16 day
+  distance: 8.32 Mpc
```

Comment:
this is likely deprecated - or at least should be.

Reply:
Yes the distance is depreciated.  I will remove it.